dlpy/tests/test_applications.py

```diff
@@ -566,3 +566,8 @@ def test_speech_recognition_1(self):
         from dlpy.applications import SpeechRecognition
         model = SpeechRecognition(self.s, n_blocks=1)
         model.print_summary()
+
```

Comment:
can you please also add a couple of more tests using different parameters than the default ones